diff --git a/src/logging.cc b/src/logging.cc
--- a/src/logging.cc
+++ b/src/logging.cc
@@ -57,6 +57,7 @@
 #include <vector>
 #include <errno.h>                   // for errno
 #include <sstream>
+#include <algorithm>
 #include "base/commandlineflags.h"        // to get the program name
 #if _WIN32
 # include "windows/glog/logging.h"
@@ -1444,7 +1445,7 @@
 #if defined(_DEBUG) && defined(_MSC_VER)
   // When debugging on windows, avoid the obnoxious dialog and make
   // it possible to continue past a LOG(FATAL) in the debugger
-  _asm int 3
+  //_asm int 3
 #else
   abort();
 #endif
